### PR TITLE
added linking to refresh-id

### PR DESCRIPTION
**Pull request by hunterprice04**

### CODE REVIEW

File: tests/main_test.py [hunk 1]
```diff
@@ -281,6 +281,56 @@ def test_invite(self):
...
+        group = response.json()['data']
+        assert self.group['name'] == group['name']
+        assert self.group['desc'] == group['desc']
+        assert self.group['restricted']['permissions']['admin_overrule_modify_transaction'] \
+               == group['restricted']['permissions']['admin_overrule_modify_transaction']
+        assert self.group['_id'] != group['_id']
+
+        # delete the group
+        response = self.do_post('/group/delete', {'id': id})
+        self.ensure_status_code_msg(response, 200, "Group successfully deleted.")
```
Comment (andreicozma1) [inline tests/main_test.py:332]: can we put these all in the teardown fixture and remove them from the end of all the functions?
For ex, if self.group exists then delete that group, etc

**Review by andreicozma1 — CHANGES_REQUESTED** [review]

File: tests/main_test.py (hunk 1 above)
Comment (andreicozma1) [reply tests/main_test.py:332]: those fixtures are meant to do this exact kind of cleanup after test case functions are run

Comment (hunterprice04) [reply tests/main_test.py:332]: These functions are test specific, in order to properly test we need a clean group each time. Adding this to the teardown function will only delete it at the end of the tests

Comment (hunterprice04) [reply tests/main_test.py:332]: cant get this to work at the moment and its slightly scope creep, if absolutely required we can make a separate issue for it

**Review by andreicozma1 — APPROVED** [review]

**Pull request merged**